project/src/main/java/com/google/sps/data/FakeSearchRequestGenerator.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.data;
+
+import com.google.maps.GeoApiContext;
+import com.google.maps.TextSearchRequest;
+
+public class FakeSearchRequestGenerator implements SearchRequestGenerator {
+
+    // The entry point for a Google GEO API request.
+    private GeoApiContext context;
+
+    /**
+     * FakeSearchRequestGenerator constructor.
+     *
+     * @param geoApiContext the GeoApiContext used for all Google GEO API requests
+     */
+    public FakeSearchRequestGenerator(GeoApiContext geoApiContext) {
+        this.context = geoApiContext;
```

Comment:
Same question..

Reply:
This is so the contex is ready to be passed to the FakeSearchRequest constructor on every create() call.